ui/desktop/app/components/credentials-panel/index.hbs

```diff
@@ -24,39 +24,25 @@
 {{#each @credentials as |credential|}}
   {{#if this.isRawApiVisible}}
     <div class='raw-credential-panel-body'>
-      <div class='raw-credential-header'>
-        <div class='raw-credential-title'>
-          <div class='credential-name hds-typography-display-200'>
-            {{#if credential.source.name}}
-              {{credential.source.name}}
-            {{else}}
-              {{credential.source.id}}
-            {{/if}}
-          </div>
-          <Hds::Copy::Button
-            @text='Copy'
-            @isIconOnly={{true}}
-            @textToCopy={{raw-json credential.rawCredential}}
-          />
-        </div>
-        <div class='hds-typography-body-100'>
+      <Hds::CodeBlock
+        @language='json'
+        @value={{raw-json credential.rawCredential}}
+        @hasLineNumbers={{false}}
+        @hasCopyButton={{true}}
+        data-test='raw-secret'
```

Comment:
Could we update this to be `data-test-raw-secret`? We normally use `data-test='xxx'` if we need `xxx` to be dynamic.